### PR TITLE
Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by vietanhduong**
Migrate CI from Travis CI to GitHub Actions

- [x] Lint check 
- [x] Test
- [x] Build kybernetwork/reserve-data image 
- [x] Build kybernetwork/reserve-data-gateway image 
- [x] Build kybernetwork/reserve-data-setting image 
- [ ] Notify to AWX Server
- [x] Deploy API Docs to GCS

### CODE REVIEW

File: .github/workflows/reserve-data.yaml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+          rm -rf /tmp/.buildx-cache
+          mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+
+  # docker build and push kybernetwork/reserve-data-setting
+  # accept branchs: v3-develop, develop, v3-stagion, staging, master, github-action (just for test)
+  build-setting:
+    needs:
+      - prepare
+      - test
+    if: contains(fromJson('["v3-develop", "develop", "v3-staging", "staging", "master", "github-action"]'), needs.prepare.outputs.current_branch)
```
Comment (secmask) [inline .github/workflows/reserve-data.yaml:202]: no v3-master?

Comment (vietanhduong) [reply .github/workflows/reserve-data.yaml:202]: `v3-master` does not appear in `.travis.yaml` so I omitted this branch.

File: .github/workflows/reserve-data.yaml [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+        shell: bash
+        id: current_branch
+        run: |
+          BRANCH="${GITHUB_REF#refs/heads/}"
+          echo "::set-output name=value::$BRANCH"
+      - name: Get Tag
+        shell: bash
+        id: tag
+        run: |
+          TAG="$(echo $GITHUB_SHA | head -c7)"
```
Comment (vietthang207) [inline .github/workflows/reserve-data.yaml:27]: Please rename this to avoid confusion with git tag! Maybe commit_hash or something else.

**Review by vietthang207 — APPROVED** [review]
LGTM

**Pull request merged**